.github/scripts/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN dnf -y --setopt=tsflags=nodocs update \
  && dnf clean all
 
 # gdb and valgrind are part of final image as they can be used as debug options for Skupper
-RUN dnf -y --setopt=tsflags=nodocs install glibc cyrus-sasl-lib libuuid openssl gettext hostname iputils libwebsockets-devel libnghttp2 gdb valgrind && dnf clean all
+RUN dnf -y --setopt=tsflags=nodocs install glibc cyrus-sasl-lib cyrus-sasl-plain cyrus-sasl-gssapi libuuid openssl gettext hostname iputils libwebsockets-devel libnghttp2 gdb valgrind && dnf clean all
```

Comment:
I would not do this here as well. This PR should just be about using the sys.executable in the Python code and removing the use of py_string from the relevant test files.

Reply:
removed during rebase onto main